### PR TITLE
Timer

### PR DESCRIPTION
**Pull request by hhamud**
I have created a basic timer component that only counts down to 0 until the auction ends. I have also added an addMinutes function so that I could test the timer.

Please check over it for any mistakes and any comments on how I can improve will be greatly welcomed.

### CODE REVIEW

File: src/data/auction.ts [hunk 1]
```diff
@@ -55,7 +58,7 @@ export function generateAuctionData(unixTimestamp: number): Auction[] {
     {
       id: 'simulation',
       startBlock: addHours(dateUTC, -55).unix(),
-      endBlock: addHours(dateUTC, +2).unix(),
+      endBlock: addMinutes(dateUTC, +1).unix(),
```
Comment (adamazad) [inline src/data/auction.ts:61]: Can we think of a case where the investor wants to observe the IDO before they decide to place a bid?

File: src/views/Auction/components/Timer.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react'
```
Comment (adamazad) [inline src/views/Auction/components/Timer.tsx:1]: We should a coding convention. We randomly decide to prefix imports like this:

1. `// Externals` for any external module(s)
2. `// Interfaces` for internal interfaces defined in `src/interfaces`
3. `// Components` for Components
4. `// Utils` for any kind of helper functions
5. `// Layouts` for any layout Component defined in `src/layouts`

This is not fixed. I am guilty of deviating away from this. but I am happy to discuss this style further :)

**Review by adamazad — APPROVED** [review]

File: src/views/Auction/components/Timer.tsx (hunk 2 above)
Comment (hhamud) [reply src/views/Auction/components/Timer.tsx:1]: yeah that is fine. I have updated my component to fit along these lines.

File: src/data/auction.ts (hunk 1 above)
Comment (hhamud) [reply src/data/auction.ts:61]: I'll keep that in mind. Although this ties into martin's other issue about making the bids more realistic. Should we include this point at that issue as well?

**Pull request merged**